src/site-type/html.php

```diff
@@ -73,11 +68,24 @@ public function __construct() {
 	 *
 	 * [--wildcard]
 	 * : Gets wildcard SSL .
+	 *
 	 * [--type=<type>]
 	 * : Type of the site to be created. Values: html,php,wp etc.
 	 *
 	 * [--skip-status-check]
 	 * : Skips site status check.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Create html site
+	 *     $ ee site create example.com
```

Comment:
Add `--type=html` in this. If the default site type is configured to some other site-type then this will fail.